source/classes.tex

```diff
@@ -173,12 +173,14 @@
 
 $M(\mathtt{X})$ is defined as follows:
 \begin{itemize}
-\item If \tcode{X} is a non-union class type, the set $M(\mathtt{X})$ is
-empty if \tcode{X} has no (possibly inherited (Clause~\ref{class.derived}))
-non-static data members;
-otherwise, it consists of the type of the first non-static data member
-of \tcode{X} (where said member may be an anonymous union), $\mathtt{X}_0$,
-and the elements of $M(\mathtt{X}_0)$.
+\item If \tcode{X} is a non-union class type with no (possibly
+inherited (Clause~\ref{class.derived})) non-static data members, the set
+$M(\mathtt{X})$ is empty.
+
+\item If \tcode{X} is a non-union class type whose first non-static data
+member has type $\mathtt{X}_0$ (where said member may be an anonymous union),
```

Comment:
I'm using the style otherwise used locally. I'd like to go forward with this change and then:

I agree we're inconsistent here, and the other changes should probably have used \mathtt (or rather, \mcode). It seems we agree that a "T"-style placeholder for a type should be \tcode (not \placeholder) and indexing should be in math mode, with $\mcode{T}_i$ the preferred style of indexing. I'd suggest to open a new issue for exactly this and fix that aspect globally. (This should be a "no visual difference" change.)